playbooks/experiments/roles/pg_remote_db_setup/tasks/main.yaml

```diff
@@ -0,0 +1,104 @@
+---
+  - name: "Install postgres and its required dependencies"
+    yum:
+      name: "{{ item }}"
+      state: present
+    with_items:
+        - postgresql
+        - postgresql-server
+        - postgresql-contrib
+    delegate_to: "{{ pgsql_host }}"
+  - name: "Initialize the postgresql"
+    shell:
+            service postgresql initdb
+    delegate_to: "{{ pgsql_host }}"
+  - name: "Configure pgsql"
+    lineinfile:
+          dest: /var/lib/pgsql/data/postgresql.conf
+          regexp: "^#listen_address"
+          line: "listen_address: '0.0.0.0'"
+          state: present
+    delegate_to: "{{ pgsql_host }}"
+  - lineinfile:
```

Comment:
Could we have "name" for at least some of these lineinfile if they are your customizations? If this is only something which mirrors Sat6 defaultsetup, then OK